Update uswds-theme.scss references (#1866)
uswds/uswds-site: pages/design-tokens/overview.md

### Example: Tokens in settings and component Sass

> Tokens can be expressed as variables, which is how most Design System theme settings work. For instance, the following is an example of theme settings in your [USWDS settings configuration]({{ site.baseurl }}/documentation/settings). This example shows settings variables that have assigned spacing unit tokens would look:

Reviewer: ```suggestion
Tokens can be expressed as variables, which is how most Design System theme settings work. For instance, the following is an example of theme settings in your [USWDS settings configuration]({{ site.baseurl }}/documentation/settings). This example shows spacing unit tokens assigned to settings variables:
```
